lang/cs/Org.Apache.REEF.IO/packages.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If it is auto generated, leave it as is. I don't think anyone ever cares about the history of the xml header tag line.
If it is BOM issue then you can take a look at this [link](https://msdn.microsoft.com/en-us/library/cc295463.aspx), but I wouldn't care.